recipes-browser/wpewebkit/wpewebkit.inc

```diff
@@ -13,7 +13,7 @@ DEPENDS = " \
 "
 
 inherit cmake pkgconfig perlnative python3native
-inherit ${@'cmake_qt5' if 'qt5-layer' in d.getVar('BBFILE_COLLECTIONS').split() else ''}
+inherit ${@'cmake_qt5' if 'qtwpe' in d.getVar('PACKAGECONFIG').split() else ''}
```

Comment:
is this change directly related to fx the build with musl?

Reply:
I see https://github.com/Igalia/meta-webkit/pull/169 open with the same changes so I assume not related. right?

Reply:
Looks like a simple rebase oversight indeed.

Reply:
yes. yoe/mut is 'master-under-test' so I sometime have mutliple fixes. I have rebased it now. Can you take a look at this again ?